package.json

```diff
@@ -7,7 +7,7 @@
   "license": "GPL-3.0",
   "dependencies": {
     "@cypress/react": "^5.12.4",
-    "bulma": "^0.9.3",
+    "bulma": "^0.9.4",
```

Comment:
![image](https://user-images.githubusercontent.com/38065883/174601123-f4352ad3-3315-48db-b7d5-5ec5e5618dbf.png)
Use `.trim()` before adding new to do to the list, to check it's length